packages/cydran/src/log/strategy/AbstractLevelStrategy.ts

```diff
@@ -0,0 +1,150 @@
+			pender.info(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+		}
+	}
+
+	public ifWarn(logLabel: string, pender: Appender, payloadFn: () => any, moreArgs: any[]): void {
+		if (this.isWarn()) {
+			const msgBase: string = resolvemsgBase(payloadFn, logLabel, pender);
+			const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+			pender.warn(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+		}
+	}
+
+	public ifError(logLabel: string, pender: Appender, payloadFn: () => any, moreArgs: any[]): void {
+		if (this.isError()) {
+			const msgBase: string = resolvemsgBase(payloadFn, logLabel, pender);
+			const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+			pender.error(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+		}
+	}
+
+	public ifFatal(logLabel: string, pender: Appender, payloadFn: () => any, moreArgs: any[]): void {
+		if (this.isFatal()) {
+			const msgBase: string = resolvemsgBase(payloadFn, logLabel, pender);
+			const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+			pender.fatal(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+		}
+	}
+
+	public trace(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.trace(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public debug(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.debug(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public info(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.info(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public warn(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.warn(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public error(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.error(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public fatal(logLabel: string, pender: Appender, msgBase: string, moreArgs: any[]): void {
+		const lpts: LogParts = this.buildParts(msgBase, moreArgs);
+		pender.fatal(logLabel, lpts.msg2Log, lpts.e2send, lpts.wkArgs);
+	}
+
+	public isTrace(): boolean {
+		return this.isLevel(Level.TRACE);
+	}
+
+	public isDebug(): boolean {
+		return this.isLevel(Level.DEBUG);
+	}
+
+	public isInfo(): boolean {
+		return this.isLevel(Level.INFO);
+	}
+
+	public isWarn(): boolean {
+		return this.isLevel(Level.WARN);
+	}
+
+	public isError(): boolean {
+		return this.isLevel(Level.ERROR);
+	}
+
+	public isFatal(): boolean {
+		return this.isLevel(Level.FATAL);
+	}
+
+	private isLevel(lvl: Level): boolean {
+		return lvl >= this.getLevel();
```

Comment:
Comparison not needed, as this information is inherent to the active level strategy.